shared/fs/filepreview/default-view.js

```diff
@@ -55,17 +55,7 @@ const DefaultView = (props: DefaultViewProps) => (
         onClick={props.onShare}
       />
     )}
-    {isIOS ? (
```

Comment:
Why is this removed?

Reply:
There's no way this code gets reached. This is in the "default view", which is only used when we don't have a viewing component. For media types, we just display them in their dedicated components.